src/devtoolsPanel.ts

```diff
@@ -210,7 +210,7 @@ export class DevToolsPanel {
         context: vscode.ExtensionContext,
         telemetryReporter: Readonly<TelemetryReporter>,
         targetUrl: string) {
-        const column = vscode.ViewColumn.Beside;
+        const column = vscode.ViewColumn.Active;

```

Comment:
What happens when you only have a single editor window (no columns) open and you launch the elements extension? Does it just become the active tab, or does it open in a second column?

Reply:
It becomes the active tab, no empty panels are created.

Reply:
Do you think the extra column issue is more annoying than the extra steps it would take to see the code and the elements tool side by side, after this change?
I'm not sure which is a better user experience, but most twitter screenshot's i've seen suggest that people are using a split view with the tool which would take more steps to accomplish after this change.
@brendyna @hxlnt - any thoughts?

Reply:
Good point, I'm not sure about it. Most of the extensions that I've seen run in the active panel, so for consistence this seems like a good change, but as you mention user will need to drag the active code to the side panel if they want to keep a side by side. I'll leave this to @brendyna and @hxlnt

Reply:
I agree with @jalissia here, having an extra step dragging active code seems to outweigh the benefits. Is there a way to have both, as in follow this way when there is no active code only?

Reply:
We don't have a way to let the user know, but we can potentially do it. I'll abandon this PR for now :) thanks both.